### PR TITLE
[DOCS] Add descriptions to CollisionObject2D

### PR DESCRIPTION
**Pull request by brunosxs**
Need some guidance to describe the only method missing info: _input_event method

### CODE REVIEW

File: doc/classes/CollisionObject2D.xml [hunk 1]
```diff
@@ -195,6 +197,7 @@
 			<argument index="1" name="enable" type="bool">
 			</argument>
 			<description>
+				Sets one way collision of the given shape owner.
```
Comment (cbscribe) [inline doc/classes/CollisionObject2D.xml]: I think it would help to have a brief explanation of what "one way collision" means. See CollisionPolygon2D.

**Review by cbscribe — CHANGES_REQUESTED** [review]

File: doc/classes/CollisionObject2D.xml [hunk 2]
```diff
@@ -68,6 +68,7 @@
 			<argument index="0" name="owner_id" type="int">
 			</argument>
 			<description>
+				Returns [code]true[/code] if one way collision is enabled for the shape owner of given index.
```
Comment (MarianoGnu) [inline doc/classes/CollisionObject2D.xml]: you should specify what "type" stands for

File: doc/classes/CollisionObject2D.xml [hunk 3]
```diff
@@ -154,6 +155,7 @@
 			<argument index="1" name="shape_id" type="int">
 			</argument>
 			<description>
+				Returns the index of the shape at given [code]owner_id[/code] or -1 if nothing was found.
```
Comment (MarianoGnu) [inline doc/classes/CollisionObject2D.xml]: same as before, what is "type"

File: doc/classes/CollisionObject2D.xml (hunk 2 above)
Comment (brunosxs) [inline doc/classes/CollisionObject2D.xml]: The description for one way collision is really brief, so could I just copy it from CollisionPolygon2D? I'd also add it to line line 71.
Also, I'll fix the entire file to add [code] around the arguments in all methods descriptions. There are quite a few descriptions without those.

Comment (cbscribe) [reply doc/classes/CollisionObject2D.xml]: That sounds good to me.

File: doc/classes/CollisionObject2D.xml (hunk 3 above)
Comment (NathanLovato) [inline doc/classes/CollisionObject2D.xml]: Could probably rephrase it to make it clearer. I'd add a line to explain how/what you use this index for.

File: doc/classes/CollisionObject2D.xml [hunk 4]
```diff
@@ -68,6 +68,7 @@
 			<argument index="0" name="owner_id" type="int">
 			</argument>
 			<description>
+				Returns [code]true[/code] if one way collision is enabled for the shape owner of given index. Only way collision is a mode where only the edges that face up, relative to CollisionObject2D's rotation, will collide with other objects.
```
Comment (NathanLovato) [inline doc/classes/CollisionObject2D.xml]: Only way collision is a mode where only the edges that face up, relative to CollisionObject2D's rotation, will collide with other objects.
->Suggestion
`One way collision is a mode where only the edges that face up, relative to the rotation [code]CollisionObject2D[/code], will collide with other objects. Call [method shape_owner_set_one_way_collision] to activate this mode.`
